src/MediaCollections/MediaCollection.php

```diff
@@ -120,4 +120,15 @@ public function withResponsiveImages(): self
 
         return $this;
     }
+
+    public function withResponsiveImagesIf($condition): self
+    {
+        if (is_callable($condition)) {
```

Comment:
Use a ternary here, so we can lose the `else` statement.

Reply:
will do, was also thinking of adding an exception to make sure we only get this 2 types.